storybook/src/stories/Dropdown.stories.svelte

```diff
@@ -0,0 +1,62 @@
+<script context="module" lang="ts">
+import { Dropdown } from '@podman-desktop/ui-svelte';
+import { type Args, defineMeta, setTemplate, type StoryContext } from '@storybook/addon-svelte-csf';
+
+/**
+ * These are the stories for the `Dropdown` component.
+ */
+const { Story } = defineMeta({
+  component: Dropdown,
+  title: 'Dropdown',
+  tags: ['autodocs'],
+  argTypes: {
+    value: {
+      control: 'text',
+      description: 'Initial value shown in the dropdown',
+      defaultValue: '',
+    },
+    disabled: {
+      control: 'boolean',
+      description: 'Set the dropdown as being disabled',
+      defaultValue: false,
+    },
+    id: {
+      control: 'text',
+      description: 'Id of the dropdown',
+      defaultValue: undefined,
+    },
+    name: {
+      control: 'text',
+      description: 'Name of the dropdown',
```

Comment:
where the name is used, is it for just an aria-label or something else

Reply:
Just the control name, used by some playwright tests. Looks like we aren't adding it to Storybook in most other cases so I removed it.